src/peft/peft_model.py

```diff
@@ -157,6 +157,10 @@ def save_pretrained(
             save_directory (`str`):
                 Directory where the adapter model and configuration files will be saved (will be created if it does not
                 exist).
+            safe_serialization (`bool`, *optional*, defaults to `False`):
+                Whether to save model in safe tensors format.
+            save_function (`function`, *optional*, defaults to `torch.save`):
+                Specify a save function. Not compatible with save serialization.
```

Comment:
Could you please add something along the lines of: "The save_function should take as argument the state_dict and the path to save to."